core/tests/test_utils.py

```diff
@@ -433,6 +434,21 @@ def _send_post_request(
     def post_email(
             self, recipient_email, sender_email, subject, body, html_body=None,
             expect_errors=False, expected_status_int=200):
+        """Post an email from the sender to the recipient.
+
+        Args:
+            recipient_email: str. The email of the recipient.
+            sender_email: str. The email of the sender.
+            subject: str. The subject of the email.
+            body: str. The body of the email.
+            html_body: str. The HTML body of the email.
+            expect_errors: bool. Whether errors are expected.
+            expected_status_int: int. The expected status code of
+                the JSON response.
```

Comment:
I think it can accommodate more words within 80 characters. Please check!

Reply:
It cannot. I had earlier tried it. There are 84 characters:  
```bash
************* Module oppia.core.tests.test_utils
C:446, 0: Line too long (84/80) (line-too-long)
```

Reply:
Fine. Thanks for the confirmation. Can you address the previous comment as well?